modules/apps/transfer/types/codec.go

```diff
@@ -3,6 +3,7 @@ package types
 import (
 	"bytes"
 
+	"github.com/cosmos/cosmos-sdk/x/authz"
```

Comment:
Looks like there's some cosmos-sdk imports underneath the linebreak below here. We should group them!